static/scss/includes/utils.scss

```diff
@@ -1,3 +1,11 @@
 .hidden {
     display: none;
 }
+
+.u-font-metropolis {
+    font-family: $font-metropolis !important;
```

Comment:
I know we shouldn't be using `!important` too much, but it didn't work without it. Since it's an overwriting class, is it fine?

Reply:
Understood. Good for now. The linter may get angry. 😅